tensorflow_serving/model_servers/main.cc

```diff
@@ -219,6 +221,11 @@ int main(int argc, char** argv) {
   core_config.aspired_version_policy =
       std::unique_ptr<AspiredVersionPolicy>(new EagerLoadPolicy);
 
+  // Override default file polling interval
+  if (file_system_poll_wait_seconds > 1) {
```

Comment:
Please drop the if-statement. (Setting it to 0 is valid, and the FileSystemStoragePathSource does (or at least should) do validation of the argument.)